src/components/submission/feed-entry.vue

```diff
@@ -129,12 +130,27 @@ export default {
       const allDiffs = (instanceId != null)
         ? this.diffs[instanceId]
         : null;
+      return allDiffs;
+    },
+    entryDiffs() {
+      const { allDiffs } = this;
       if (!allDiffs)
         return null;
       // Filters out diffs about instanceID and deprecatedID
       return allDiffs.filter((entry) =>
         last(entry.path) !== 'instanceID' &&
         last(entry.path) !== 'deprecatedID');
+    },
+    newVersionId() {
+      const { details } = this.entry;
+      if (details == null) return null;
+      const { instanceId } = details;
+      return instanceId;
+    },
+    oldVersionId() {
+      const deprecatedIdDiff = this.allDiffs.filter((entry) => last(entry.path) === 'deprecatedID')[0];
```

Comment:
i changed this to `find()` but left the comparison as just a string comparison on the last element since array comparison with `['meta', 'deprecatedID']` seems more complicated.